src/hotspot/share/runtime/arguments.cpp

```diff
@@ -44,6 +44,7 @@
 #include "oops/oop.inline.hpp"
 #include "prims/jvmtiExport.hpp"
 #include "runtime/arguments.hpp"
+#include "runtime/crac.hpp"
```

Comment:
Are you sure about this? `crac.hpp` is added to `precompiled.hpp`, so I guess it's enough for success compilation.

Reply:
I am using for all my compilations `--disable-precompiled-headers` as I had various problems before. File `./src/hotspot/share/precompiled/precompiled.hpp` contains:
```
// Precompiled headers are turned off if the user passes
// --disable-precompiled-headers to configure.

#ifndef DONT_USE_PRECOMPILED_HEADER
```
Which explains why one should not depend on `precompiled.hpp`.

I also use `--enable-ccache` if that matters but that should not matter.

Reply:
Oh, yes, we should support enabled and disabled precompiled headers.

Reply:
Great! I'm merging this. Thanks!